<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Formulario</class>
 <widget class="QWidget" name="Formulario">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>222</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Formulario</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="1">
    <widget class="QLineEdit" name="leCantidadDeCuerdas"/>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="leMetalUsado"/>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="leMarca"/>
   </item>
   <item row="6" column="2">
    <widget class="QPushButton" name="btnAgregar">
     <property name="text">
      <string>Agregar</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="cbInstrumento"/>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="btnVerStock">
     <property name="text">
      <string>Ver Stock</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="leCantidadDeTeclas"/>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lePrecio"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lMarca">
     <property name="text">
      <string>Marca</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lPrecio">
     <property name="text">
      <string>Precio</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lCuerdas">
     <property name="text">
      <string>Cuerdas</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lMetalUsado">
     <property name="text">
      <string>Metal</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lCantidadTeclas">
     <property name="text">
      <string>Teclas</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
